src/main.cpp

```diff
@@ -1289,6 +1290,7 @@ bool SetBestChain(CTxDB& txdb, CBlock &blockNew, CBlockIndex* pindexNew) EXCLUSI
         pindexNew->nHeight,
         pindexNew->GetBlockTime(),
         blockNew.nBits);
```

Comment:
Including the uiInterface.NotifyBlocksChanged call in the ifdef breaks the GUI block counter for people who do not have std::system.

Reply:
should the `#endif` be placed on line 1286 instead?

Reply:
Yep.